quaternions.scad

```diff
@@ -327,4 +447,216 @@ function Qrot(q,p) =
 module Qrot_copies(quats) for (q=quats) Qrot(q) children();
 
 
+// Function: Q_Rotation()

```

Comment:
Nice!

Reply:
This function task could be done by Q_Slerp and Q_Matrix4 but it would be very inefficient.